cmd/nerdctl/fmtutil.go

```diff
@@ -75,7 +75,12 @@ func formatSlice(cmd *cobra.Command, x []interface{}) error {
 		for _, f := range x {
 			var b bytes.Buffer
 			if err := tmpl.Execute(&b, f); err != nil {
-				return err
+				if _, ok := err.(template.ExecError); ok {
+					// FallBack to Raw Format
```

Comment:
Why do we need this

Reply:
We need this to encode struct with `tags` (RawFormat) and execute tmpl on the  RawFormat in the case where `--format` point to tag like  `Id` and not  `ID`